### PR TITLE
[metricbeat]kubernetes: add kube-state-metrics persistentvolumes

### PR DESCRIPTION
**Pull request by odacremolbap**
kubernetes volume metrics for kube-state-metrics.
refer to:

https://github.com/kubernetes/kube-state-metrics/blob/master/docs/persistentvolume-metrics.md

related to elastic/beats#14821

### CODE REVIEW

**Comment by odacremolbap** [comment]
this needs some update based on 
- https://github.com/elastic/beats/pull/15066#pullrequestreview-331106212
- https://github.com/elastic/beats/pull/15066#pullrequestreview-331109497

File: metricbeat/module/kubernetes/state_persistentvolume/_meta/test/ksm-unit-1.8 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# HELP kube_persistentvolume_info Information about persistentvolume.
+# TYPE kube_persistentvolume_info gauge
+kube_persistentvolume_info{persistentvolume="test-pv-available",storageclass=""} 1
+# HELP kube_persistentvolume_labels Kubernetes labels converted to Prometheus labels.
+# TYPE kube_persistentvolume_labels gauge
+kube_persistentvolume_labels{label_app="mysql-server",persistentvolume="test-labeled-pv"} 1
+kube_persistentvolume_labels{persistentvolume="test-unlabeled-pv"} 1
+# HELP kube_persistentvolume_capacity_bytes Persistentvolume capacity in bytes.
+# TYPE kube_persistentvolume_capacity_bytes gauge
+kube_persistentvolume_capacity_bytes{persistentvolume="test-pv"} 5.36870912e+09
```
Comment (exekias) [inline metricbeat/module/kubernetes/state_persistentvolume/_meta/test/ksm-unit-1.8]: how come this one (`test-pv`) doesn't have an status phase?

Comment (odacremolbap) [reply metricbeat/module/kubernetes/state_persistentvolume/_meta/test/ksm-unit-1.8]: those test cases are taken from kube-state-metrics unit tests in order to test all tested cases upstream. Those are not real but created at ksm for each of the supported scenarios.

Using a (real)  resource from one of our kubernetes clusters targeted a very small subset of cases.

**Comment by odacremolbap** [comment]
Changes requested at https://github.com/elastic/beats/pull/15066#discussion_r357049776 have been also applied to this metricset.

reviews are again welcomed

**Comment by andresrc** [comment]
I see #15066 backported, but not this one. Is it?

**Comment by odacremolbap** [comment]
testing notes: https://github.com/odacremolbap/beats/blob/b2ddea1a5a1952ad99e90966499226404b3329b9/metricbeat/module/kubernetes/state_persistentvolume/README.md

**Pull request merged**